pyvista/plotting/widgets.py

```diff
@@ -1210,9 +1210,9 @@ def add_mesh_threshold(
         preference : str, optional
             When ``mesh.n_points == mesh.n_cells`` and setting
             scalars, this parameter sets how the scalars will be
-            mapped to the mesh.  Default ``'points'``, causes the
-            scalars will be associated with the mesh points.  Can be
-            either ``'points'`` or ``'cells'``.
+            mapped to the mesh.  Default ``'cell'``, causes the
```

Comment:
I noticed that this function (and maybe a few others) uses `preference='cell'` by default, and most or all of the rest use `preference='point'`. Can/should we change this? This can subtly change downstream code with not warning.

Reply:
This should be `'point'`, but changed in a separate PR. Agree this is non standard behavior.

Reply:
We have an issue for that https://github.com/pyvista/pyvista/issues/2515

Reply:
Thanks @MatthewFlamm. I thought we had a PR changing that for filters and this was somehow left out. Haven't had the time to look for it yet :)